README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
+The ARG performs most of its work in terms of Ada Interpretations (AIs), which are documents that describe an identified issue with the Ada Reference Manual, or which propose enhancements to the Ada programming language.  Such issues may be identified within the ARG itself, or by the wider Ada community.
+
+At ARG meetings, the ARG will deliberate on active AIs, and vote on the progression of each AI, ultimately towards acceptance, or no action. Discussion and votes are available via the [ARG minutes](http://ada-auth.org/arg-minutes.html).
+
+AIs that are being or have been deliberated on by the ARG can be found [here](https://arg.adaic.org/ada-interpretations).
+
+As the ARG deliberates on open AIs, AIs will be updated with changes and additional discussion, or may lead to the establishment of a Language Study Group if more intensive development work is required.
+
+At the present time, AIs are being stored in the Rich Text Format, with an aim to eventually develop tooling to easily convert it into many formats. RTF was selected due to its maturity, stability, and generality. The ARG will continue to review the AI formats in the future.
+
+An AI template can be found in the root of this repository.
+
+# A new way to engage the Ada user community
+
+Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements for consideration by the ARG.  
 
 This repository, in conjunction with a new [website](https://arg.adaic.org/home), has been created to simplify the process of providing Ada community input to the ARG, and to provide more visibility into the ARG process.
 
 ## An accessible medium for discussion
 
 This repository, when compared with the sister [website](https://arg.adaic.org/home), is geared towards supporting real-time discussions on proposed features or changes to the Ada language through Github's Issues feature.
 
+Users that have comments or proposals are encouraged to open a new issue in this repository to drive initial discussion, including input from members of the ARG.
+
 Active issues can be browsed and discussed with broad accessibility and transparency.
 
+## Getting started
 
-# Language Study Groups
+The best way to get started is to browse open issues to get a sense of how issues are structured, and to see if you find any existing issues that interest you. Consider speaking with supporters within that discussion about proposing a Language Study Group to take the issue further.
 
-Language Study Groups can be formed by the community to work transparently towards a formal proposal (Ada Interpretation/AI) for consideration by ISO/IEC JTC 1/SC 22/WG 9's Ada Rapporteur Group (ARG).
+If there is an issue that you have not found discussed, please feel welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
```

Comment:
You are using "issue" in two different senses in this one sentence.  I would suggest you say instead something like:

> If there is a *topic* that you have not yet seen discussed, please feel welcome to open a new issue.

Reply:
Fixed.